src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -1,13 +1,15 @@
 import * as React from 'react';
+import PropTypes from 'prop-types';
 import {Animated, Image, Pressable, StyleSheet, Text, View} from 'react-native';
 
-import MemoIc_Checklist from '../../../assets/icons/Ic_Checklist';
+import IconChecklist from '../../../assets/icons/Ic_Checklist';
 import {DEFAULT_PROFILE_PIC_PATH} from '../../../utils/constants';
 import {COLORS} from '../../../utils/theme';
 
-const ItemUser = ({photo, username, followed, onPress, userid}) => {
+const ItemUser = ({photo, username, followed, onPress, userid, isAnon}) => {
   const [isSelect, setIsSelect] = React.useState(false);
   const followIconFadeAnimation = React.useRef(new Animated.Value(0)).current;
+  const anonColorCondition = isAnon ? COLORS.anon_primary_15 : COLORS.signed_primary_15;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `followed` prop seems to be an array as it's used with the `includes` method. It would be better to initialize `isSelect` state based on whether `userid` is included in `followed` or not. This way, you won't need to run this check inside a useEffect and can avoid unnecessary re-renders.
```diff
- const [isSelect, setIsSelect] = React.useState(false);
+ const [isSelect, setIsSelect] = React.useState(followed.includes(userid));
```
Remove the useEffect that checks if `userid` is in `followed`.


<!-- This is an auto-generated comment by OSS CodeRabbit -->